cfn-resources/network-container/cmd/resource/resource.go

```diff
@@ -81,24 +81,32 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	if err != nil {
 		if res.StatusCode == http.StatusConflict {
 			_, _ = logger.Debugf("Container already exists for this group. Try return existing container. err: %v", err)
-			containers, _, err2 := client.Containers.ListAll(context.Background(), *projectID, nil)
-			if err2 != nil {
+			containers, _, err := client.Containers.ListAll(context.Background(), *projectID, nil)
+			if err != nil {
 				_, _ = logger.Debugf("Error Containers.ListAll err:%v", err)
 				return handler.ProgressEvent{}, fmt.Errorf("error Containers.ListAll err:%v", err)
 			}
-			_, _ = logger.Debugf("containers:%v", containers)
-			if containers != nil {
-				first := containers[0]
```

Comment:
issue - Create Operation: if a container in the same region already exists, we were returning the first container returned by `ListAll` instead of returning the ID of the container in the same region.